.github/actions/spell-check/patterns.txt

```diff
@@ -10,6 +10,9 @@ data:[a-zA-Z=;,/0-9+-]+
 "Lorem[^"]+?\."
 TestCase\("[^"]+"
 
+# Test line with hexadecimal colors
+\[DataRow\("[0-9A-F]{6}", \d{3}, \d{3}, \d{3}\)\]
```

Comment:
You can fix this in various ways...

One approach is to merge the various flavors (you can test w/ a [regex tester](https://regex101.com/), or on your fork in a different branch):
```
\[DataRow\("[0-9A-F]{6}", (?:"[A-Z]\d+"|\d{3}(?:\.\d|))(?:, \d{3}(?:\.\d|)){2}\)\]
```
Or you could have multiple expressions for the individual cases -- I think one expression is reasonable as long as people can more or less read it. (You could also relax it further, but DataRow is used by others in the repository, so I'm not advocating ignoring everyone else's uses of it.) -- I've compressed it a bit because it's a bit repetitive. But it'd depend on who's reviewing it as to whether the `{2}` is a worthwhile compression.

Reply:
Thanks, but i prefer two separate patterns, that make it easier to read